pg-auto-failover.spec

```diff
@@ -82,6 +82,9 @@ PATH=%{pginstdir}/bin:$PATH
 
 
 %changelog
+* Thu May 7 2020 - Jelte Fennema <Jelte.Fennema@microsoft.com> 1.3.1-1
+- Official 1.3.1 release of pg_auto_failover
```

Comment:
I would have wanted to see a consistent item in here such as:
```
  - Official release for 1.3.1
```

Reply:
I think it's better with the name of the tool in there too. 
I've modified `update_os_package.pl` to do this automatically for new versions:
https://github.com/citusdata/tools/pull/69/files#diff-99ba4ed771cad68c7bae27b53c0c4aa3R91